org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -10,10 +10,20 @@
     <method name="turnOff"/>
     
     <!-- Get the current ambient temperature -->
-    <method name="getCurrentTemperature">
+    <method name="getAmbientTemperature">
+      <!-- Temperature scale:  'f' for Fahrenheit. 'c' for celsius -->
+      <arg name="scale" type="s" direction="in" />
```

Comment:
From the documentation and in general, i saw that 'temperature scale' usage.. so I preferred it over 'units' .

Taqi /Ravi ?

---
In reply to: [68283058](https://github.com/openT2T/translators/pull/22#discussion_r68283058) [](ancestors = 68283058)